app/Services/Spotify/OAuth/SpotifyTokenManager.php

```
@@ -0,0 +1,181 @@
+     * Get access token, automatically refreshing if needed.
+     *
+     * @return string|null The access token or null if not available
+     */
+    public function getAccessToken(): ?string
+    {
+        // Check if token needs refresh
+        if ($this->shouldRefreshToken()) {
+            $this->refreshToken();
+        }
+
+        return Cache::get(self::ACCESS_TOKEN_KEY);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'getAccessToken'. (undefined)

(StaticAccess)